### ISSUE TITLE
Implement new widget template for AI analysis

### ISSUE THREAD

**Issue by hursevich** [zebrunner/zebrunner#356]
As far as we've release AI feature we need to visualize results on dashboards.
I assume we need PIE chart with parametarized period 24h / 1 week / 1 month that shows distribution by issue type.

SQL sample: 
select count(t.id), sl.name, DATE(t.created_at)
from demo.tests t 
inner join demo.stacktrace_labels sl
on t.id = sl.test_id
where sl.reporter = 'AI'
group by DATE(t.created_at), sl.name

**Comment by vdelendik** [comment]
blocked by the reporting categorization feature which shoudl be implemented for CE

**Comment by vdelendik** [comment]
closing as won't fix as AI/ML failure analysis is not an option for CE

**Comment by dhreben** [comment]
Won't fix. Closed
